test/models/work_test.rb

```diff
@@ -0,0 +1,96 @@
+require "test_helper"
+
+describe Work do
+  before do
+    # Arrange
+    @work = Work.new(category: "book", title: "The Cat in the Hat", creator: "Dr. Seuss", publication_year: 1957, description: "anthropomorphic cat")
+    # username = params[:user][:username]
+    user = User.create(username: "test_user")
+    @login_user = User.all.first
+  end
+
+  describe "relations" do
+      it "has many votes" do
+        parable = works(:parable)
+        vote1 = votes(:vote1)
+        vote2 = votes(:vote2)
+
+        parable.votes << vote1
+        parable.votes << vote2
```

Comment:
You should remove these lines because these relationships are already created in your fixtures.